src/Files.App/ViewModels/Settings/PreferencesViewModel.cs

```diff
@@ -605,9 +605,16 @@ public string Text
 				if (Path == "Home")
 					return "Home".GetLocalizedResource();
 
-				return (Path == CommonPaths.RecycleBinPath)
-					   ? ApplicationData.Current.LocalSettings.Values.Get("RecycleBin_Title", "Recycle Bin")
-					   : Path;
+				if (Path.Equals(CommonPaths.RecycleBinPath, StringComparison.OrdinalIgnoreCase))
```

Comment:
Can you use the helper?

Reply:
This is actually different from the helper I've just pushed. I'll add another one